CHANGELOG.md

```diff
@@ -1,5 +1,9 @@
 # Braintree iOS Drop-in SDK - Release Notes
 
+## Unreleased
+
+* Require Braintree ~> 4.29
```

Comment:
Should we mention why? For Pay with Venmo to work on iOS 13?

Reply:
Personally, I don't think we need to call it out. We can always look at the changelog for `braintree_ios` version 4.29.0.

Reply:
Right! Makes sense to not duplicate it, like we've discussed before. Thanks for the reminder.